src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -1356,6 +1356,14 @@ class MacroAssembler: public Assembler {
     vmfle_vv(vd, vs1, vs2, vm);
   }
 
+  inline void vmsltu_vi(VectorRegister Vd, VectorRegister Vs2, int32_t imm, VectorMask vm = unmasked) {
+    vmsleu_vi(Vd, Vs2, imm-1, vm);
+  }
+
+  inline void vmsgeu_vi(VectorRegister Vd, VectorRegister Vs2, int32_t imm, VectorMask vm = unmasked) {
```

Comment:
Same comments as `vmsltu_vi ` above.

Reply:
It's used at https://github.com/openjdk/jdk/pull/16562/files#diff-97f199af6d1c8c17b2fa4f50eb1bbc0081858cc59a899f32792a2d31f933ccc4R3945.